pkg/sdk/external_tables.go

```diff
@@ -0,0 +1,472 @@
+package sdk
+
+import (
+	"context"
+	"database/sql"
+	"time"
+)
+
+var (
+	_ ExternalTables = (*externalTables)(nil)
+	_ validatable    = (*CreateExternalTableOptions)(nil)
```

Comment:
hmm, you still have this line (and other) in this file and in external_tables_validations, why?